service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunner.java

```diff
@@ -120,6 +129,64 @@ public void createDatabaseWithDbRole(
         aksNamespace);
   }
 
+  public void pgDumpDatabase(
```

Comment:
do you have any tests to make sure that the pg_dump that you created is what you exception? same for when you restore?

Reply:
no, and I'm not sure how I would do that - do you have a suggestion?

this isn't exactly the same thing, but we do have a plan to generate an md5sum of the dump file, and verify that it's the same when we restore (to let us know that the dump file hadn't been altered). but that will be part of the overall db clone flight.

Reply:
the way we do this in WDS today is we have a docker container that has a db that we can actually take a backup off, and then we have a backup file checked into the repo that we compare the dump to. 

for the restore, we take the checked in dump and restore, then actually run a query to check that the schema was set up correctly and WDS has the expected number of rows that is expected.